README.md

```diff
@@ -400,7 +400,7 @@ This section lists the currently supported algorithms. If you would like support
 
 Supported private key formats:
 - RSA in `RSA PRIVATE KEY`
-- RSA in `OPENSSH PRIVATE KEY` (`openssh-key-v1`)
+- RSA, ECDSA in `OPENSSH PRIVATE KEY` (`openssh-key-v1`)
```

Comment:
I don't know if ECDSA keys in SSH can actually use other curves than the 3 emitted by `ssh-keygen` and implemented in this PR. I can explicitly call them out if you wish or just leave it generic like it is here.

Reply:
This is good.

A bit lower, you can add the specific algorithms under "Supported client key algorithms:".

Reply:
Ah I missed that section, have added it there.